dt-demo/dimensions/src/test/java/com/datatorrent/demos/dimensions/telecom/HBaseReader.java

```diff
@@ -0,0 +1,67 @@
+package com.datatorrent.demos.dimensions.telecom;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.CellUtil;
+import org.apache.hadoop.hbase.client.HTable;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datatorrent.api.Context.OperatorContext;
+import com.datatorrent.contrib.hbase.HBaseStore;
+import com.datatorrent.demos.dimensions.telecom.conf.DataWarehouseConfig;
+import com.datatorrent.demos.dimensions.telecom.conf.EnrichedCDRHBaseConfig;
+import com.datatorrent.demos.dimensions.telecom.model.EnrichedCDR;
+
+public class HBaseReader<T>
```

Comment:
Actually, there should be a way to walk around by create a sub class of CustomerService